Templates/E-Commerce/Magento/Magento to Zoho CRM - Products/long-description.md

```diff
@@ -0,0 +1,28 @@
+This template process retrieves product data from Adobe Commerce / Magento and either inserts or updates each product in Zoho CRM.
+
+![Template](assets/Magento_to_Zoho_CRM_-_Products.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Magento user is eligible to obtain an access token from Magento.
+- The Zoho CRM client ID and client secret are accessible.
+- The domain for the Zoho instance (e.g. EU or USA) is identified.
+- A Zoho CRM refresh token with an adequate scope has been generated using the "Zoho CRM - Exchange grant token for refresh token" template. For this template, the token should be configured with the scopes `ZohoCRM.modules.products.READ` and `ZohoCRM.modules.products.CREATE`.
+
+# Implementation and Usage Notes
+
+Before retrieving the Magento product data, an access token used for authorizing the request is obtained from the Magento API. 
+
+Accessing the Zoho CRM API also requires an access token, which is obtained with a separate refresh token provided in the process variables. To generate this refresh token, you can use the corresponding "Zoho CRM - Exchange grant token for refresh token" template.
```

Comment:
In this paragraph, should we add the information about the refresh tokens not expiring? With this wording, it gives me the vibes that I need to provide a new refresh token every time I use this template.